li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -168,6 +173,11 @@
                 atLeast(1),
                 Importance.MEDIUM,
                 MAX_POLL_RECORDS_DOC)
+        .define(TOPIC_CREATION_POLL_INTERVAL_MS_CONFIG,
+                Type.INT,
+                300000,
```

Comment:
as i said above - why does this need to be independant from max.metadata.age ?

Reply:
will use metadata.max.age.ms